RadixWallet/Features/DisplayEntitiesControlledByMnemonic/DisplayEntitiesControlledByMnemonic+View.swift

```diff
@@ -94,7 +96,9 @@ extension DisplayEntitiesControlledByMnemonic {
 					}
 
 					if viewStore.accounts.isEmpty {
-						NoContentView("Hidden Accounts only")
+						if viewStore.hasHiddenAccounts {
+							NoContentView("Hidden Accounts only.")
```

Comment:
Is this an officially sanctioned string, or something temporary? It's permanent please add `FIXME: Strings`!

Reply:
Good catch!